### DISCUSSION TITLE
LDAP, OAuth2, SAML etc.

### DISCUSSION THREAD

**Discussion by durchgraf** [stumpapp/stump#242] [Ideas]
Hey,

Are there plans to implement LDAP or other SSO Solutions in the future? :)

**Comment by aaronleopold** [comment]
Hey 👋 

No concrete plans at the moment, but it's a great enhancement. Would you like me to convert this discussion to an issue?

**Reply by aaronleopold** [reply]
https://github.com/stumpapp/stump/issues/246

**Comment by aaronleopold** [comment]
Stump supports [OIDC](https://www.stumpapp.dev/guides/access-control/oidc) with the latest releases
